common.py

```diff
@@ -19,6 +19,11 @@ def enum_convert(enum_type: Type[E], value: Any) -> E:
             return enum_type[value]
         except KeyError:
             raise ValueError(f"Cannot convert {value} to {enum_type}")
+    elif isinstance(value, int):
+        try:
+            return enum_type(value)
+        except ValueError:
+            raise ValueError(f"Cannot convert {value} to {enum_type}")
```

Comment:
this part should move to the patch that first introduced `enum_convert()`. The change is also not specifically needed by this patch.